extensions/adobe/experience/marketo/account-marketo.schema.json

```diff
@@ -0,0 +1,124 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/account-marketo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/account"],
+  "meta:extends": ["https://ns.adobe.com/xdm/context/account-crm"],
+  "description": "Marketo specific Account fields.",
+  "definitions": {
+    "account-marketo": {
+      "properties": {
+        "marketo:accountID": {
+          "title": "Marketo Account Identifier",
+          "description": "Marketo account identifier.",
+          "type": "string"
+        },
+        "marketo:instanceID": {
+          "title": "Instance Identifier",
+          "description": "Instance identifier.",
+          "type": "string"
+        },
+        "marketo:org": {
+          "properties": {
+            "marketo:notes": {
+              "title": "Account Notes",
+              "description": "Account notes.",
+              "type": "string"
+            },
+            "marketo:accountType": {
+              "title": "Account Type",
+              "description": "Account type.",
+              "type": "string"
+            },
+            "marketo:primaryInterestID": {
+              "title": "Primary Interest Id",
+              "description": "Primary interest id.",
+              "type": "string"
+            },
+            "marketo:private": {
+              "title": "Private Flag",
+              "description": "Private flag.",
+              "type": "boolean"
+            },
+            "marketo:additionalFields": {
+              "title": "Additional Fields",
+              "description": "Additional fields.",
+              "type": "string"
+            }
+          }
+        },
+        "marketo:namedAccount": {
+          "properties": {
```

Comment:
As note previously - better descriptions

Reply:
Please refer to the first comment